plugins/by-name/nvim-ufo/default.nix

```
@@ -130,4 +130,22 @@ lib.nixvim.plugins.mkNeovimPlugin {
         end
       '';
   };
+
+  extraOptions = {
+    setupLspCapabilities = lib.nixvim.options.mkEnabledOption "setup LSP capabilities for nvim-ufo";
+  };
+
+  extraConfig =
+    cfg:
+    lib.mkIf cfg.enable {
```

```suggestion
  extraConfig = {
```

No need to check `cfg.enable` as that is already done by mkNeovimPlugin

---

done